src/ape_pm/compiler.py

```diff
@@ -1,12 +1,21 @@
 import json
+from json import JSONDecodeError
 from pathlib import Path
 from typing import List, Optional, Set
 
 from ethpm_types import ContractType
 
 from ape.api import CompilerAPI
+from ape.exceptions import CompilerError
+from ape.logging import logger
 from ape.utils import get_relative_path
 
+KNOWN_JSON_IGNORE_LIST = (
+    "package.json",
+    "package-lock.json",
+    "tsconfig.json",
+)
+
```

Comment:
Hmm, would this make sense as a global ignore via config file?

e.g,

```yaml
compiler:
    ignore-files:
    - package.json
    - package-lock.json
    - tsconfig.json
```

Reply:
Done. I made it a separate PR and will strip it out of this one, thanks!
https://github.com/ApeWorX/ape/pull/1151